### DISCUSSION TITLE
Pressing enter key on non Autocomplete suggestions

### DISCUSSION THREAD

**Discussion by mikebhart** [algolia/instantsearch#4516] [General]
I'm using autocomplete and I want to be able to:-

Select an item from suggestions and do something. Im Ok with this.

I want to be able to take the user somewhere else If the user types in a search string and just presses the enter key and ignores the suggestions.  

The problem I have is that it forces the user to select the first suggestion even though I might not like it and just want to press enter and take the user somewhere else.

How do I get the autocomplete to stop forcing me to select the first suggested item.

**Comment by Haroenv** [comment]
please open an issue with reproduction on https://github.com/algolia/autocomplete.js

**Comment by mikebhart** [comment]
im using the instantsearch autocomplete, this is my code and why doesnt these properties work:-
	
const autocomplete = connectAutocomplete(({ indices, refine, widgetParams }, isFirstRendering) => {
		const { container, onSelectChange } = widgetParams;

		if (isFirstRendering) {
			container.html('<select id="ais-autocomplete"></select>');

			container.find('select').selectize({
				options: [],
				valueField: 'course_title',
				labelField: 'course_title',
				highlight: false,
				onType: refine,
				onBlur() {
					refine(this.getValue());
				},
				score() {
					return function () {
						return 1;
					};
				},
				onChange(value) {
					onSelectChange(this.options[value].course_url);
					refine(value);
				},
			});

			return;
		}

		const [select] = container.find('select');

		indices.forEach((index) => {
			select.selectize.clearOptions();
			index.results.hits.forEach((h) => select.selectize.addOption(h));
			select.selectize.refreshOptions(select.selectize.isOpen);
		});
	});

	const suggestions = instantsearch({
		indexName: coursesIndexName,
		searchClient,
	});

	suggestions.addWidgets([
		autocomplete({
			autoselect: false,
			openOnFocus: true,
			container: $('#autocomplete'),
			onSelectChange(value) {
				window.location.href = value;
			},
		}),
	]);

	suggestions.start();
	searchCoursesLanding.start();
